engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSender.java

```diff
@@ -83,9 +84,17 @@ public void sendEmail(User user, Event event, TemplateInstance subject, Template
         LocalDateTime start = LocalDateTime.now(UTC);
 
         Timer.Sample processedTimer = Timer.start(registry);
+
         EventType eventType = event.getEventType();
-        String bundleName = eventType.getApplication().getBundle().getName();
-        String applicationName = eventType.getApplication().getName();
+        String bundleName = "NA";
+        String applicationName = "NA";
+        if (eventType != null) {
+            bundleName = eventType.getApplication().getBundle().getName();
+            applicationName = eventType.getApplication().getName();
```

Comment:
Is this going to work ? This is a stateless session, from what I remember it throws an exception - but I could be wrong :sweat:

Reply:
nevermind, it's equal to the previous code